docker-compose.yml

```diff
@@ -1,5 +1,40 @@
 services:
-  hca:
+  # Ollama per CPU-Profil
+  hca-ollama-cpu:
+    image: ollama/ollama
+    container_name: ollama
+    ports:
+      - "11434:11434"
+    volumes:
+      - ollama:/root/.ollama
```

Comment:
We should move away away from using the root dir.
/etc/ollama or /var/etc/ollama might be better options
TBD